### PR TITLE
modules to use time_in()

### PR DESCRIPTION
**Pull request by tobes**
Get modules to use `time_in()`

### CODE REVIEW

File: py3status/modules/aws_bill.py [hunk 1]
```diff
@@ -95,7 +94,7 @@ def _get_bill_amount(self):
 
     def aws_bill(self):
         response = {
-            'cached_until': time() + self.cache_timeout,
+            'cached_until': self.pys.time_in(self.cache_timeout),
```
Comment (ultrabug) [inline py3status/modules/aws_bill.py]: py3

File: py3status/modules/bitcoin_price.py [hunk 2]
```diff
@@ -73,7 +72,7 @@ def _get_price(self, data, market, field):
 
     def get_rate(self):
         response = {
-            'cached_until': time() + self.cache_timeout,
+            'cached_until': self.time_in(self.cache_timeout),
```
Comment (ultrabug) [inline py3status/modules/bitcoin_price.py]: py3

File: py3status/modules/dropboxd_status.py [hunk 3]
```diff
@@ -38,7 +37,7 @@ class Py3status:
     format = 'Dropbox: {}'
 
     def dropbox(self):
-        response = {'cached_until': time() + self.cache_timeout}
+        response = {'cached_until': self.time_in(self.cache_timeout)}
```
Comment (ultrabug) [inline py3status/modules/dropboxd_status.py]: py3

Comment (tobes) [inline py3status/modules/dropboxd_status.py]: clever you

File: py3status/modules/process_status.py [hunk 4]
```diff
@@ -51,7 +50,7 @@ def process_status(self):
             (color, text) = self._get_text()
 
         response = {
-            'cached_until': time() + self.cache_timeout,
+            'cached_until': self.y3.time_in(self.cache_timeout),
```
Comment (ultrabug) [inline py3status/modules/process_status.py]: py3

**Comment by ultrabug** [comment]
awesome :)

**Pull request merged**